admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -746,6 +746,14 @@ var form = (function() {
   function switchLanguage(iso_code) {
     $('div.translations.tabbable > div > div.translation-field:not(.translation-label-' + iso_code + ')').removeClass('show active');
     $('div.translations.tabbable > div > div.translation-field.translation-label-' + iso_code).addClass('show active');
+    //Force refresh tinymce editors
+    $('div.translations.tabbable > div > div.translation-field.translation-label-' + iso_code + ' textarea.autoload_rte').each(function(index, textarea) {
```

Comment:
I see this line twice on my screen 😛 dont you want to make reuse it to avoid duplicated content ?

Also this way you'll escape the "Line too long" of @PierreRambaud 😄

Reply:
Do you mean @matks ? :trollface:

Reply:
OK ^^ fixed